File: modules/core/src/main/java/kr/entree/enderwand/command/sender/ConsoleSender.java
package kr.entree.enderwand.command.sender;

/**
 * Created by JunHyung Lim on 2020-02-04
 */
public class ConsoleSender implements Sender {
    public static final ConsoleSender INSTANCE = new ConsoleSender();

    private ConsoleSender() {
    }

    @Override
    public void sendMessage(Object message) {
        System.out.println(message);
    }

    @Override
    public boolean hasPermission(Object node) {
        return true;
    }

    @Override
    public boolean isOp() {
        return true;
    }
}

File: modules/core-kotlin/src/main/kotlin/kr/entree/enderwand/command/sender/ConsoleSender.kt
package kr.entree.enderwand.command.sender

/**
 * Created by JunHyung Lim on 2019-12-18
 */
class ConsoleSender : Sender {
    override fun tell(message: Any) {
        println(message)
    }

    override fun tellError(errorMessage: Any) {
        tell(errorMessage)
    }

    override fun hasPermission(node: String) = true

    override fun isOp() = true
}